web-client/public/js/graph.js

```diff
@@ -1070,7 +1071,10 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
 
             // For whatever reason the arc collapses to a point if the beginning
             // and ending points of the arc are the same, so kludge it.
-                        x2 = d.source.x + d.source.textWidth / 1.2;
+                        if(d.source.textWidth > 135){
```

Comment:
Are you running the linter? (`npm run lint`) This line should be flagged by it. If it isn't, then we should look into whatever setting in the linter will ensure that this is properly spaced. i.e.,
```javascript
if (d.source.textWidth > 135) {
```

Reply:
That was my bad! Changed it.

Reply:
On the side note, there are 325 errors when I run lint.